virtual_workstation_setup.sh

```diff
@@ -107,6 +116,20 @@ setup_gdb() {
     popd
 }
 
+setup_undodb() {
+    local evg_username=$(evg_user)
+    if [ -z $evg_username ]; then
+        echo "UndoDB: can't figure out what your SSO username is. Set the 'UNDO_user' environment variable to your Okta username in your shell's rc file before using UndoDB"
+        echo "ex: export UNDO_user='john.doe'"
+        return
+    fi
+
+    echo "\nexport UNDO_user='$evg_username'" >> ~/.bashrc
```

Comment:
I think we use `.bash_profile` instead of `.bashrc` because it's an interactive session. Maybe just put it in both to be safe?

Reply:
.bash_profile should be sourcing .bashrc, so I've gone and done that. If memory serves correct, RHEL's /etc/bash_profile actually does this by default but debian doesn't. It should be perfectly safe to do this.

I very much do not want to write duplicate information to multiple places.